protocols/kad/CHANGELOG.md

```diff
@@ -4,9 +4,11 @@
   See [PR 4270].
 - Reduce noise of "remote supports our protocol" log.
   See [PR 4278].
+- Depend on `quick-protobuf-codec` crate for codec logic.
```

Comment:
Version `0.44.4` has already been released. Please make a new entry above! See https://github.com/libp2p/rust-libp2p/blob/master/CONTRIBUTING.md#write-changelog-entries-for-user-facing-changes and the linked release docs! :)

Reply:
You will also need to bump the version in the `Cargo.toml`!

Reply:
> You will also need to bump the version in the `Cargo.toml`!

And also the referenced version in the root `Cargo.toml` because we use workspace inheritance to sync the versions of our crates across the workspace.

Reply:
Done! Would `0.44.5` be the next logical bump?

Reply:
> Done! Would `0.44.5` be the next logical bump?

Yes!